liana/method/_Method.py

```diff
@@ -249,6 +254,11 @@ def __call__(self,
                                resource=resource,
                                expr_prop=expr_prop,
                                min_cells=min_cells,
+                               met_est_resource_name=None,
```

Comment:
Hmmm, instead of doing this, you should make `MetaboliteMethod` a child of Method